internal/store/json.go

```diff
@@ -29,8 +30,9 @@ type workspaceV0 map[string]blueprint.Blueprint
 // It contains all the information necessary to generate the inputs for the job, as
 // well as the job's state.
 type composeV0 struct {
-	Blueprint   *blueprint.Blueprint `json:"blueprint"`
-	ImageBuilds []imageBuildV0       `json:"image_builds"`
+	Blueprint   *blueprint.Blueprint           `json:"blueprint"`
+	ImageBuilds []imageBuildV0                 `json:"image_builds"`
+	PackageSets map[string][]rpmmd.PackageSpec `json:"package_sets"`
```

Comment:
As suggested elsewhere: should we make this only be the `packages` set, so we don't make the names of the sets API, nor the build-root++